brave/brave-core
# [Android] custom filter list settings

Resolves https://github.com/brave/brave-browser/issues/30174



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open settings
2. Go to Brave Shields & privacy
3. Go to Content Filtering
4. Add Custom filter list url
5. Check if it is working same as desktop

https://github.com/brave/brave-core/assets/11678818/a0a63fe7-bb46-4281-941c-b0a020961723
